### PR TITLE
:recycle: refactor: pool page

### PR DESCRIPTION
**Pull request by atilasousa**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206801469137424

### CODE REVIEW

File: views/pools/header/action-group.tsx [hunk 1]
```diff
@@ -21,39 +14,22 @@ const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
         display={['none', 'none', 'none', 'flex']}
       >
         <TextField
-          Prefix={
-            <Box height="1.25rem" width="1.25rem">
-              <SearchSVG maxHeight="100%" maxWidth="100%" width="100%" />
-            </Box>
-          }
-          placeholder="Search"
+          fontFamily="Proto"
```
Comment (MarioBatalha) [inline views/pools/header/action-group.tsx:17]: Change this input to button

File: views/pools/header/search-mobile.tsx [hunk 2]
```diff
@@ -12,20 +12,25 @@ const SearchMobile: FC<SearchMobileProps> = ({
   return (
     <Box
       gap="m"
-      px="xs"
       width="100%"
       display={showSearchView ? ['flex', 'flex', 'flex', 'none'] : 'none'}
     >
       <TextField
```
Comment (MarioBatalha) [inline views/pools/header/search-mobile.tsx:18]: Here too

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/pools/header/action-group.tsx [hunk 3]
```diff
@@ -20,39 +19,28 @@ const ActionGroup: FC<ActionGroupProps> = ({ showSearchView }) => {
...
-        />
         <Button
           py="s"
-          bg="onSurface"
-          color="surface"
-          variant="filled"
-          onClick={handleCreatePool}
+          color="onSurface"
+          variant="tonal"
+          bg="#ffffff14"
```
Comment (MarioBatalha) [inline views/pools/header/action-group.tsx:26]: Use bg  from theme

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by KipandaJr — APPROVED** [review]

**Review by MarioBatalha — APPROVED** [review]

File: views/pools/pool-filter/index.tsx [hunk 4]
```diff
@@ -6,39 +6,37 @@ import { FilterSVG } from '@/svg';
...
-      alignItems={['unset', 'unset', 'unset', 'center']}
-      flexDirection={['column', 'column', 'column', 'row']}
-      justifyContent={['center', 'center', 'center', 'flex-start']}
+      pb="s"
+      gap="2xs"
+      alignItems="center"
+      flexDirection="row"
+      justifyContent="flex-start"
+      gridTemplateColumns="1fr 1fr 1fr"
+      display={['grid', 'flex', 'flex', 'flex', 'flex']}
```
Comment (git-marcopitra) [inline views/pools/pool-filter/index.tsx:20]: remove the unneeded `flex`

File: views/pools/pool-filter/index.tsx [hunk 5]
```diff
@@ -6,39 +6,37 @@ import { FilterSVG } from '@/svg';
...
         height="2.5rem"
-        display="flex"
-        justifyContent="center"
-        alignItems="center"
         color="onSurface"
         borderRadius="xs"
+        alignItems="center"
         flexDirection="column"
+        justifyContent="center"
+        display={['none', 'none', 'inline-flex', 'inline-flex', 'inline-flex']}
```
Comment (git-marcopitra) [inline views/pools/pool-filter/index.tsx:31]: remove unneeded `inline-flex`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/pools/pool-filter/index.tsx (hunk 4 above)
Comment (atilasousa) [reply views/pools/pool-filter/index.tsx:20]: ??

File: views/pools/pool-filter/index.tsx (hunk 5 above)
Comment (atilasousa) [reply views/pools/pool-filter/index.tsx:31]: unneeded why??

File: views/pools/header/index.tsx [hunk 6]
```diff
@@ -21,11 +21,13 @@ const Header: FC<HeaderProps> = ({ currentTab, setTab }) => {
...
   return (
     <Box
-      display="flex"
+      gap="s"
+      width="100%"
       flexWrap="wrap"
       overflow="none"
       alignItems="center"
       justifyContent="space-between"
+      display={'flex'}
```
Comment (atilasousa) [inline views/pools/header/index.tsx:30]: why {}?

Comment (KipandaJr) [reply views/pools/header/index.tsx:30]: accident

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**